### PR TITLE
add rubocop

### PR DESCRIPTION
**Pull request by yuchan**
I introduced [rubocop](https://github.com/bbatsov/rubocop), which is a tool to check coding style.

Simply,

```
bundle install
bundle exec rubocop -a
```

your code would be [ruby-styled](https://github.com/bbatsov/ruby-style-guide).

Let's talk about it at the meeting on someday.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,34 +11,34 @@
...
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141012124824) do
-  create_table "quizzes", force: true do |t|
-    t.string   "question"
-    t.string   "answer"
-    t.datetime "created_at"
-    t.datetime "updated_at"
-    t.integer  "user_id"
+ActiveRecord::Schema.define(version: 20_141_012_124_824) do
```
Comment (not001praween001) [inline db/schema.rb]: ?

Comment (yuchan) [inline db/schema.rb]: exactly. this is weird.

**Comment by not001praween001** [comment]
Thank you. 
But i don't think that it needs on our app now. 
The most important is resolving the callback bug. 
Pending to the meeting discussion.

**Comment by yuchan** [comment]
yes, of course.
I am looking forward to seeing callback fixed. :)

**Comment by not001praween001** [comment]
I am looking forward to seeing everyone including Toshiteru San to do it together.
The first one is TOSHITERU SA~N

**Comment by not001praween001** [comment]
(In addition, I have idea about the GitHub coding comment field! I think if github comment field has Like button or emotion sticker post feature it become more fun!!!! _o_  lol)

File: db/schema.rb (hunk 1 above)
Comment (not001praween001) [inline db/schema.rb]: ^o^

**Pull request closed without merging**